pkg/vsphere/vm/vm.go

```diff
@@ -565,32 +565,32 @@ func (vm *VirtualMachine) DatastoreReference(ctx context.Context) ([]types.Manag
 	return mvm.Datastore, nil
 }
 
-// GetVCHUpdateStatus tells if an upgrade/update has already been started based on the UpdateInProgress flag in ExtraConfig
-// If error != nil, VCH upgrade/update will exit; so the value of the VCH upgrade/update status is not important
-func (vm *VirtualMachine) GetVCHUpdateStatus(ctx context.Context) (bool, error) {
+// VCHUpgradeStatus tells if an upgrade/configure has already been started based on the UpgradeInProgress flag in ExtraConfig
+// It returns the error if the vm operation does not succeed
+func (vm *VirtualMachine) VCHUpgradeStatus(ctx context.Context) (bool, error) {
```

Comment:
this method will be reused by upgrade and configure, so might still use update as name